<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MetadataDialog</class>
    <widget class="QDialog" name="MetadataDialog">
        <property name="windowTitle">
            <string>Song Metadata</string>
        </property>
        <layout class="QFormLayout">
            <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Title*</string></property></widget></item>
            <item row="0" column="1"><widget class="QLineEdit" name="editTitle"/></item>
            <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Artist*</string></property></widget></item>
            <item row="1" column="1"><widget class="QLineEdit" name="editArtist"/></item>
            <item row="2" column="0"><widget class="QLabel"><property name="text"><string>Album</string></property></widget></item>
            <item row="2" column="1"><widget class="QLineEdit" name="editAlbum"/></item>
            <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Year</string></property></widget></item>
            <item row="3" column="1"><widget class="QSpinBox" name="spinYear"><property name="minimum"><number>0</number></property><property name="maximum"><number>3000</number></property></widget></item>
            <item row="4" column="0"><widget class="QLabel"><property name="text"><string>Genre</string></property></widget></item>
            <item row="4" column="1"><widget class="QLineEdit" name="editGenre"/></item>
            <item row="5" column="0" colspan="2">
                <layout class="QHBoxLayout">
                    <item><spacer name="spacer"><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
                    <item><widget class="QPushButton" name="btnCancel"><property name="text"><string>Cancel</string></property></widget></item>
                    <item><widget class="QPushButton" name="btnOK"><property name="text"><string>OK</string></property></widget></item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>